README.md

```diff
@@ -13,14 +13,15 @@ usage: installvm.py [-h] --host-ip HOST_IP --host-name HOST_NAME --host-gw
                     [--hmc-userid HMC_USERID] [--hmc-password HMC_PASSWORD]
                     [--hmc-profile HMC_PROFILE] [--ksargs KSARGS]
                     [--showcleanup SHOWCLEANUP] --distro DISTRO
-                    [--set-boot-order SET_BOOT_ORDER]
+                    [--set-boot-order SET_BOOT_ORDER] --gsa-server GSA_SERVER
 
 optional arguments:
   -h, --help            show this help message and exit
   --distro DISTRO       distro to be installed ex: rhel_7.4le_alpa,
                         sles_11sp3_beta
   --set-boot-order SET_BOOT_ORDER
                         yes/True to set the boot disk order
+  --gsa-server GSA_SERVER GSA server domain name for SSL certification
```

Comment:
just be have generic as server where user need to have SSL enable to access the system